appinventor/buildserver/src/com/google/appinventor/buildserver/BuildServer.java

```diff
@@ -73,6 +75,7 @@
 @Path("/buildserver")
 public class BuildServer {
   private ProjectBuilder projectBuilder = new ProjectBuilder(statReporter);
+  private String hostname = null;
```

Comment:
Alternatively, you could assign `hostname = getEtcHostname()` here so that it executes when the class is instantiated, and then reference `hostname` on line 277.